src/components/EventCard/EventCard.js

```diff
@@ -6,9 +6,17 @@ const EventCard = ({ name, date, venue, city, state, image, id }) => {
   const [year, month, day] = date.split("-");
   const dateDisplay =
     month && day && year ? `${month}/${day}/${year}` : `Multi-day Event`;
-
+console.log(`&&&`, image);
   return (
-    <div className="event-card">
+    <div className="event-card"
+      style={{
```

Comment:
Curious why there are inline styles here?

Reply:
This was the best way we found to make the background image dynamic and speak to each event's image.